src/madetomeasure_api/routes/subscribers.clj

```diff
@@ -29,8 +29,8 @@
 (defn create-subscribers [request]
   (let [subscribers (extract-subscribers request)]
     (try
-      (apply clojure.java.jdbc/insert! db/db-spec :subscribers subscribers)
-      (present-response/success subscribers)
+      (let [insertions (apply clojure.java.jdbc/insert! db/db-spec :subscribers subscribers)]
```

Comment:
I agree though. I don't like the dropping into java. Probably cause I hate java.